datasync/chaindatafetcher/kafka/config.go

```diff
@@ -42,10 +44,14 @@ const (
 	DefaultRequiredAcks         = 1
 	DefaultSegmentSizeBytes     = 1000000 // 1 MB
 	DefaultMaxMessageNumber     = 100     // max number of messages in buffer
+	DefaultKafkaMessageVersion  = MsgVersion1_0
+	DefaultProducerIdPrefix     = "producer-"
 )
 
 type KafkaConfig struct {
 	SaramaConfig         *sarama.Config `json:"-"` // kafka client configurations.
+	MsgVersion           string         // MsgVersion is the version of Kafka message.
+	ProducerId           string         // ProducerId is for the identification of the message publisher.
```

Comment:
`String` method doesn't print new fields.

Reply:
The method is updated. Thanks.